dev_tools/scripts/create_new_algorithm.sh

```diff
@@ -24,9 +24,9 @@ function print_help {
 function list_sources {
     for type in solver preconditioner matrix factorization
     do
-        for i in $(ls $GINKGO_ROOT_DIR/core/$type/*.cpp)
+	for i in "${GINKGO_ROOT_DIR}/core/${type}/"*.cpp;
```

Comment:
globs should also be evaluated in quotes, I think?
```suggestion
        for i in "${GINKGO_ROOT_DIR}/core/${type}/*.cpp";
```